### DISCUSSION TITLE
500: INTERNAL_SERVER_ERROR after deploying Django project on Vercel. Please help!

### DISCUSSION THREAD

**Discussion by niharikaasinghal** [vercel/community#2629] [Help]
### Summary

I am trying to deploy my Django project which is a simple blog website and displays blogs stored in the django admin. My project contains static files. After deploying, this is the error i am getting-
![Screenshot (3520)](https://github.com/vercel/community/assets/110163537/0d181cc7-a3e4-438a-907e-cb33d17e9207)

### Steps to Reproduce

Here are some screenshots of settings.py , requirements.txt and wsgi.py files-

settings.py-

![Screenshot (3521)](https://github.com/vercel/community/assets/110163537/de83fee1-4d28-4281-93b8-7c903c80b55c)

![Screenshot (3522)](https://github.com/vercel/community/assets/110163537/70399988-97ee-4120-a5d9-355cf643af06)

requirements.txt-

![Screenshot (3523)](https://github.com/vercel/community/assets/110163537/980c101e-cfca-488e-8d70-4857ca7965da)

wsgi.py-

![Screenshot (3524)](https://github.com/vercel/community/assets/110163537/43f13948-0c56-4713-ad97-7836d658a488)

**Comment by azihassan** [comment]
Which Python version are you using ? Make sure it's 3.9 since Django 4 requires 3.8+ and if I'm not mistaking, Vercel supports 3.9
